### DISCUSSION TITLE
Block Unicode symbols for ChatFilter

### DISCUSSION THREAD

**Discussion by Snele** [Zrips/CMI#7022] [Feature Requests]
I would like it to be possible to prohibit unicode characters in ChatFilter literally in one line, for example: 
Unicode: false

**Comment by MrCrazys** [comment]
Why not just use a RegEx expression? It's a kind of unnecessary option. Also, please use the Discussions tab for feature requests next time (https://github.com/Zrips/CMI/discussions).

**Reply by Snele** [reply]
RegEx does not allow to fully block all unicode characters

**Comment by CloudeLecaw** [comment]
na adding this unicode is to a regex expression is to complicated fir many.

so a simple
Unicode: true/false settng would be a nice userfriendly setting for many users.

well nothing speaks aggainst adding a chatfilter with regex too if you set this setting to false. ;)

**Comment by Rexmen1** [comment]
yea would love to see this, It is really complicated for many server owners...
